file-thread/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "file-thread"
+version = "0.1.0"
+authors = ["Anton Patrushev <apatrushev@gmail.com>"]
+edition = "2018"
+
+[[bin]]
+name = "main"
+path = "src/main.rs"
+
+[[bin]]
+name = "tap"
+path = "src/tap.rs"
+
+[dependencies]
+actix = "0.7"
+bytes = "*"
```

Comment:
`0.1`

Reply:
Mine idea was that actix itself depends on bytes crate, so this code can rely on its dependency restriction. If it is a bad practice I will change it.

Reply:
if you use `*` while actix uses `0.1` you gonna use two different crates when `0.2` appears